src/test/java/com/snowflake/kafka/connector/streaming/iceberg/TestJsons.java

```diff
@@ -0,0 +1,144 @@
+package com.snowflake.kafka.connector.streaming.iceberg;
+
+/**
+ * Class to provide Iceberg schema evolution tests with schemas and payload. It decreases the size
+ * of the test file .
+ */
+class TestJsons {
```

Comment:
maybe it would be worth moving the schemas to separate json files and in this class only read from the files?